app/views/articles/_sidebar_nav.html.erb

```diff
@@ -45,29 +45,51 @@
 </nav>
 
 <nav class="mb-6" aria-label="Secondary sidebar nav">
-  <% if user_signed_in? %>
+<nav class="mb-6" aria-label="Secondary sidebar nav">
+  <% if user_signed_in? && current_user.following_tags_count >= 1 %>
     <header class="p-2 pr-0 flex items-center justify-between">
       <h3 class="crayons-subtitle-3">My Tags</h3>
       <a href="/dashboard/following_tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Customize tag priority" title="Customize tag priority">
         <%= inline_svg_tag("cog.svg", aria: true, class: "crayons-icon") %>
       </a>
     </header>
     <div id="sidebar-nav-followed-tags" class="overflow-y-auto mb-2" style="max-height: 42vh;"></div>
-  <% else %>
-    <h3 class="crayons-subtitle-3 p-2">Popular Tags</h3>
-    <div id="sidebar-nav-default-tags" class="overflow-y-auto" style="max-height: 42vh">
-      <% Tag.where(supported: true).order(hotness_score: :desc).limit(30).pluck(:id, :name).each do |tag_array| %>
-        <div class="sidebar-nav-element" id="default-sidebar-element-<%= tag_array.second %>">
-          <a class="crayons-link crayons-link--block py-2 px-2" href="<%= tag_path(tag_array.second) %>">
-            #<%= tag_array.second %>
-          </a>
-          <a class="follow-action-button sidebar-nav-link-follow crayons-btn crayons-btn--s"
-            href="#" id="sidebar-nav-link-follow-<%= tag_array.second %>"
-            data-info='{"id":<%= tag_array.first %>,"className":"Tag"}'>
-            Follow
-          </a>
+    <% elsif user_signed_in? %>
+      <header class="p-2 pr-0 flex items-center justify-between">
+        <h3 class="crayons-subtitle-3">Explore Tags</h3>
+        <a href="/tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Tags Overview" title="Tags Overview">
+          <%= inline_svg_tag("cog.svg", aria: true, class: "crayons-icon") %>
```

Comment:
The cog icon strikes me as a bit confusing in this context, but I think a different icon something we can follow up with at a later point, personally.

Reply:
@jacobherrington: what are your thoughts on linking "Explore Tags" to the `/tags` page and just taking the `cog_svg` out entirely? It would look like **this when not hovered over**:
<img width="201" alt="Screen Shot 2020-10-22 at 12 21 52 PM" src="https://user-images.githubusercontent.com/32834804/96913660-2d572080-1461-11eb-9ab5-f6b62e079047.png">
**And this when hovered over**:
<img width="195" alt="Screen Shot 2020-10-22 at 12 22 15 PM" src="https://user-images.githubusercontent.com/32834804/96913707-3a740f80-1461-11eb-8f4c-9894e48f624c.png">

Reply:
I don't have a strong opinion, but this does feel a little more natural to me!

Reply:
Changed @jacobherrington!